clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
```

Comment:
This function should probably have some docstring.

As for predicates - you should really adopt the Emacs conventions as you're writing Emacs Lisp, not Clojure. @magnars is one of the few Elisp hackers who uses `?`, but in `dash.el` even he added the traditional names as aliases. I profoundly dislike violating established conventions for no good reason.